resources/js/components/GistComments.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+    <div>
+        <gist-comment
+            v-for="comment in comments"
+            :key="comment.id"
+            :gist-id="gistId"
+            :comment="comment"></gist-comment>
+        <div v-if="loading">
+            Loading comments...
+        </div>
+    </div>
+</template>
+
+<script>
+    export default {
+        props: {
+            gistAuthor: String,
+            gistId: String,
+        },
+        data: function () {
+            return {
+                comments: [],
+                loading: true,
+            };
+        },
+        mounted() {
+            this.loadComments();
+        },
+        methods: {
+            loadComments() {
+                axios.get(`/${this.gistAuthor}/${this.gistId}/comments.json`)
```

Comment:
We could use some named routes and Ziggy here.

Reply:
yesssurrr